protocol/avdl/rekey.avdl

```diff
@@ -32,6 +38,12 @@ protocol rekey {
     array<ProblemTLF> tlfs;
   }
 
+  record ProblemSetDevices {
+    User user;
+    KID kid;
+    array<ProblemTLFDevices> tlfs;
```

Comment:
In other words, this is what I had in mind:

```record ProblemSetDevices {
   ProblemSet problemSet;
   array<Device> devices;
}```

Then electron can perform the mapping, and we don't list the same device over and over again...